src/components/Swap/Toolbar/index.tsx

```diff
@@ -37,6 +37,28 @@ const ToolbarRow = styled(Row)`
   padding: 0 1em;
 `
 
+const Context = createContext<{
```

Comment:
i'd name this something more specific like `ToolbarContext`

Reply:
i think we prefer this name elsewhere in the codebase if the Context is not exported.